src/test/java/api/requests/RequestsAPILoanHistoryTests.java

```diff
@@ -66,21 +68,26 @@ public void creatingRecallRequestChangesTheOpenLoanForTheSameItem()
   }
 
   @Test
-  public void creatingPageRequestDoesNotChangeTheOpenLoanForSameItem()
+  public void failureCreatingPageRequestDoesNotChangeTheOpenLoanForSameItem()
```

Comment:
Given that it is now not possible to create a page request when an item is checked out, it should not be possible that there is an open loan either. 

Hence, I'm not sure what this test is now trying to demonstrate, and I think it could be clearer to remove it. What do you think?

Reply:
I agree. I'll remove it. This test doesn't add much value in terms of documentation.